### ISSUE TITLE
turn location on automatically is still greyed out 

### ISSUE THREAD

**Issue by ybrhue** [Seva-coder/Finder#43]
Rooted redmi note 8 pro.
I run the command in termux as follows:
su pm grant ru.seva.finder android.permission.WRITE_SECURE_SETTINGS

And granted the permission.
However it is still greyed out in the menu!

**Comment by daarkwulf** [comment]
yeah i did the same and every setting. this app is not able to trun on automatically i found one with this feature and it working all time. and it is this https://github.com/xfarrow/locatemydevice     also this locate my device is having many but same feature specially callme and wifi on off feature. which is really imp. plus it auto turn on location by code. 

so now we have to install two apps. one is our finder and locatemydevice. and to use it first it we have to send code regarding the turn on location automatically and then will use our code of finder to send us locations . but both apps works great and satisfy the need .     

one more thing i found was findmydevice on fdroid app which was having feature of sending time to time location and taking pic and upload to server but really it will not work unless phone has internet and we all know if phone got lost the person will surely turn off the internet first. so only sms service will be left out in that case

and if he removed the sim card then surely there is only way to locate is through imei.

but i was also thinking if there is any app which will work as simcard even if the real sim card will be removed. that gonna be best untill we can able to locate by imei which is mostly done by govt only. 

keep update us guys.
